src/main.ts

```diff
@@ -64,7 +65,7 @@ export async function createProject(options: Options) {
 
   try {
     await tasks.run();
-    await renderOutroMessage(options);
+    renderOutroMessage(options);
```

Comment:
why you removed await here btw? TS shows error here
<img width="761" alt="image" src="https://github.com/scaffold-eth/create-eth/assets/25638585/b2f5118e-4281-49ce-aae0-d9f9926a92e5">